fusillade/clouddirectory.py

```diff
@@ -18,7 +18,7 @@
 
 project_arn = "arn:aws:clouddirectory:us-east-1:861229788715:"  # TODO move to config.py
 cd_client = boto3.client("clouddirectory")
-
+iam = boto3.client("iam")
```

Comment:
If this code is executed from a Lambda, consider putting this behind an `lru_cache`

Reply:
Adding todos